### DISCUSSION TITLE
TreeGenerator TDSM is necessary

### DISCUSSION THREAD

**Discussion by svenjahoffmann** [UMEP-dev/UMEP#519] [General]
Dear UMEP Community,
I try to use the TreeGenerator in the Toolbox UMEP for processing. I have a canopy DSM and a point shapefile with new trees. With the TreeGenerator I want to add new trees to the current canopy DSM. I get the Error message: "No valid vegetation TDSM raster layer is selected. Both CDSM and TDSM must be selected if merging with existing". I already tried TreeGenerator months ago and it worked without TDSM, why do I need it now? I only want an updated CDSM, I don't want to update the TDSM (as it does not even exist). A few months ago I used the "normal" TreeGenerator Tool, not the one in the Toolbox for processing. But these days it's not possible anymore to use the "normal" one, there is the message: "This tool has moved to UMEP for processing, available via the QGIS Plugin Manager. Visit our online manual for more information".

Thanks for your help!!

**Comment by biglimp** [comment]
I suggest that you submit an issue so that we can fix it properly. In the meantime you can do a workaround. Use the raster calculator and create a TDSM by multiplying your CDSM with 0.25. Then you can run the TreeGenerator and update your CDSM with new trees.
